### PR TITLE
Azcopy fix

### PR DESCRIPTION
**Pull request by billwert**

### CODE REVIEW

File: scripts/azcopy.py [hunk 1]
```diff
@@ -73,18 +73,37 @@ def upload_files(self, search_path: str):
...
+                                               filename)
+                    os.rename(file, newname)
+
+                renamed_files = glob(path.join(
+                                        get_artifacts_directory(),
+                                        '**',
+                                        '*perf-lab-report.json'), recursive=True)
+
+
+                dirname = path.dirname(renamed_files[0])
```
Comment (adiaaida) [inline scripts/azcopy.py]: Does this ever get used?

Comment (adiaaida) [reply scripts/azcopy.py]: Oh, nevermind. I found it.

File: scripts/benchmarks_ci.py [hunk 2]
```diff
@@ -259,8 +260,8 @@ def __main(args: list) -> int:
 
         benchview.run_scripts(args, verbose, BENCHMARKS_CSPROJ)
 
-        if(args.upload_to_perflab_container): 
-            AzCopy.upload_results(args.upload_to_perflab_container, verbose)
+        if args.upload_to_perflab_container: 
+            AzCopy.upload_results('', verbose=verbose)
```
Comment (jorive) [inline scripts/benchmarks_ci.py]: Is empty string intended?

File: scripts/benchmarks_ci.py [hunk 3]
```diff
@@ -147,7 +147,8 @@ def __is_valid_datetime(dt: str) -> str:
     parser.add_argument('--upload-to-perflab-container',
         dest="upload_to_perflab_container",
         required=False,
-        help="Container to upload perf lab results to."
+        help="Container to upload perf lab results to.",
```
Comment (jorive) [inline scripts/benchmarks_ci.py]: Do we need default value here?

File: scripts/benchmarks_ci.py (hunk 2 above)
Comment (billwert) [reply scripts/benchmarks_ci.py]: Yes.

File: scripts/benchmarks_ci.py (hunk 3 above)
Comment (billwert) [reply scripts/benchmarks_ci.py]: good catch - need to update help. (though no one is ever actually going to use this but us.)

**Pull request merged**